custom_components/better_thermostat/models/models.py

```diff
@@ -54,8 +52,7 @@ def get_device_model(self):
 
 def convert_outbound_states(self, hvac_mode):
 	"""Convert HA state to outbound thermostat state."""
-	get_device_model(self)
-	
+
```

Comment:
shouldn't we make sure that we have the device model fetched, at this stage?

Reply:
Not needed, we load the device model at startup, if it fails we exit with an error